scripts/lib/workspace_command.py

```
@@ -0,0 +1,778 @@
+        and os.environ.get("WORKSPACE_COMMAND_PATH") == expected
+    ):
+        return
+    raise SystemExit(2)
+
+
+def env_value(name: str, default: str = "") -> str:
+    """Return a stripped environment value used by promoted Python commands."""
+    return os.environ.get(name, default).strip()
+
+
+def require_env(name: str, usage: str | None = None) -> str:
```

Ruff (0.16.1)

[warning] Unused function argument: `usage`

(ARG001)